### PR TITLE
[log4cplus/muparser/nmslib] Update to the latest version

### PR DESCRIPTION
**Pull request by NancyLi1013**
Update ports to latest version

Related #13536

Note: llibzip has been updated in master branch, so I revert the changes for `libzip `in this PR.

### CODE REVIEW

File: ports/nmslib/portfile.cmake [hunk 1]
```diff
@@ -1,29 +1,22 @@
...
-    set(WITH_EXTRAS ON)
-endif()
-
 # TODO: check SSE and AVX avability and set corresponding tags
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}/similarity_search
     PREFER_NINJA
-    OPTIONS
-        -DWITH_EXTRAS=${WITH_EXTRAS}
+    OPTIONS ${FEATURE_OPTIONS}
```
Comment (ras0219-msft) [inline ports/nmslib/portfile.cmake:19]: `FEATURE_OPTIONS` is not defined in this file.
```suggestion
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

File: ports/nmslib/portfile.cmake (hunk 1 above)
Comment (NancyLi1013) [reply ports/nmslib/portfile.cmake:19]: Thanks for your reminder. I have removed the feature, so this option is also not needed now. Sorry, I forgot to remove this.

**Review by PhoebeHui — APPROVED** [review]
LGTM

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**